app/assets/css/app.css

```diff
@@ -71,8 +71,12 @@ body,
 }
 
 input[type='checkbox'] {
-  margin-top: 1px;
   vertical-align: middle;
+  -moz-transform: scale(1.8);
+  -webkit-transform: scale(1.8);
+  -o-transform: scale(1.8);
+  margin-left: 2px;
+  margin-bottom: 5px;
```

Comment:
This change is too global, not sure where there is a checkbox that won't play nice with this. I would apply this rule only to checkboxes in the datatables, which is basically `.md-checkbox input[type='checkbox']`.

A cleaner solution is to give the checkbox static height and width like the wrapping element. `.md-checkbox` has height and width of 16px. applying the same to the checkbox would fix the problem IMO

Reply:
Will have a look at this, thanks for the tip